docs/getting-started.ts

```diff
@@ -34,7 +36,8 @@ async function main(): Promise<void> {
   })
 
   /* To store the CTYPE on the blockchain, you have to call: */
-  Kilt.connect(DEFAULT_WS_ADDRESS)
+  Kilt.config({ host: NODE_URL })
+  Kilt.connect(ConfigService.getNodeAddress())
```

Comment:
isn't the point of the config service that the endpoint would be fetched implicitly ? I'd move `ConfigService.getNodeAddress()` into `Kilt.connect()`